pom.xml

```diff
@@ -264,28 +212,27 @@
           </execution>
         </executions>
       </plugin>
-<!--
+
       <plugin>
         <groupId>org.openapitools</groupId>
         <artifactId>openapi-generator-maven-plugin</artifactId>
         <version>${openapi-generator.version}</version>
         <executions>
           <execution>
-            <id>inn-reach</id>
+            <id>DTO Generation</id>
             <goals>
               <goal>generate</goal>
             </goals>
             <configuration>
-              <inputSpec>${i2nreach.yaml.file}</inputSpec>
+              <inputSpec>${openapi.input.file}</inputSpec>
               <output>${project.build.directory}/generated-sources</output>
               <generatorName>spring</generatorName>
-              <modelPackage>${project.groupId}.edge.i2nreach.domain.dto</modelPackage>
-              <apiPackage>${project.groupId}.edge.i2nreach.rest.resource</apiPackage>
+              <modelPackage>${project.groupId}.edge.dto</modelPackage>
```

Comment:
Looking on another spring-related modules there are used naming convention like:
``
*.<module-semantic-related-name>.domain.dto
*.<module-semantic-related-name>.rest.resource
``
for example. Does it make sense to use here similar approach? If so, some kind of `'*.ir.domain.dto"` and `"*.ir.rest.resource"` or somethoing similar could be used here. How do you think?